<?xml version="1.0"?>
<launch>
	<group ns="cyberpod">

		<node name="momdp_node" pkg="segway_sim" type="momdp_node.py" output="screen">
			<param name="expFlag" value="1"/>
			<param name="option" value="3"/>
		</node>
	
	</group>
</launch>
